doc/sphinx/language/core/conversion.rst

```diff
@@ -79,16 +67,28 @@ reduction is called δ-reduction and shows as follows.
    --------------
    E[Γ] ⊢ c~\triangleright_δ~t
 
+:term:`Delta-reduction <delta-reduction>` only unfolds constants that are
+marked :gdef:`transparent`.  :gdef:`Opaque <opaque>` is the opposite of
+transparent; :term:`delta-reduction` doesn't unfold opaque constants.
+
+ι-reduction
+~~~~~~~~~~~
 
-.. _zeta-reduction:
+A specific conversion rule is associated with the inductive objects in
+the global environment. We shall give later on (see Section
+:ref:`Well-formed-inductive-definitions`) the precise rules but it
+just says that a destructor applied to an object built from a
+constructor behaves as expected. This reduction is called
+:gdef:`ι-reduction <iota-reduction>`
+and is more precisely studied in :cite:`Moh93,Wer94`.
 
 ζ-reduction
 ~~~~~~~~~~~
 
-Coq allows also to remove local definitions occurring in terms by
-replacing the defined variable by its value. The declaration being
-destroyed, this reduction differs from δ-reduction. It is called
-ζ-reduction and shows as follows.
+:gdef:`ζ-reduction <zeta-reduction>` removes local definitions in terms by
```

Comment:
I think calling these definitions let-in definitions rather than local definitions is more explicit and thus clearer:

```suggestion
:gdef:`ζ-reduction <zeta-reduction>` removes :ref:`let-in definitions <let-in>` in terms by
```